tools/test-backend

```diff
@@ -187,7 +187,10 @@ class ZulipInternetBlockedError(Exception):
 
 
 def main() -> None:
-    default_parallel = os.cpu_count()
+    # WORKAROUND: at n=64, I regularly get an error in
+    # zerver.tests.test_example.TestDevelopmentEmailsLog.test_generate_emails
+    # os.cpu_count() can return None, so shutup mypy with a sensible default
+    default_parallel = min(55, int(os.cpu_count() or 1))
```

Comment:
I think we get to drop this hunk, since we fixed that bug? :)

Reply:
DONE :-)